Shopify/cli
# Add a schema to the capabilities attribute of extensions

Related: https://github.com/Shopify/cli/pull/455#discussion_r980469922

### WHY are these changes introduced?
The `capabilities` section of the UI extensions' configuration file has no schema in the CLI. Because of it, the serving logic ended up having [some logic](https://github.com/Shopify/cli/pull/455/files/cacc3f232d5d88608d31a64c65c418fa25a43ec4#diff-e5f8a674cef6431ad1097c19b9112388b3267bcf2f8db6939b38e1d01effeb10R25) that assumes the schema and its defaults. This is information that shouldn't be part of the serving business logic so I'm codifying it in the schema.

### WHAT is this pull request doing?
I'm adding a schema and defaults to the `capabilities` attribute of the UI extension configuration file.

### How to test your changes?
The `product-subscription` UI extension of the `fixtures/app` directory has the `network_access` capability set to true. Run `dev` with that project passing `--verbose` and confirm that `networkAccess` is returned as `true` from the extensions endpoint.

### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
